### ISSUE TITLE
About path: target: '/media/4T/Dizzy/AIM/AIM_datasets/DIV2K_train_HR'

### ISSUE THREAD

**Issue by Gptao** [ShuhangGu/DASR#3]
is it the original DIV2K_train_HR?

**Comment by DizzyYunxuan** [comment]
Yes, it is the original DIV2K HR. You can find more data descriptions in AIM2019 official pages. https://competitions.codalab.org/competitions/20164

**Comment by Gptao** [comment]
👌 thank you very much!



            
            
                
                    
                            
                                
                            
                            
                                taoguangpin
                            
                    
                        
                            
                                    邮箱：taoguangpin@qq.com
                            
                        
                
            
        Signature is customized by Netease Mail Master


            On 09/15/2020 23:27, DizzyYunxuan wrote: 
Yes, it is the original DIV2K HR. You can find more data descriptions in AIM2019 official pages. https://competitions.codalab.org/competitions/20164

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/ShuhangGu/DASR/issues/3#issuecomment-692792106",
"url": "https://github.com/ShuhangGu/DASR/issues/3#issuecomment-692792106",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by Gptao** [comment]
By the way, HR images are used to get high-frequency information. Can I replace this with 800s  in flickr2k? Because I'm doing synthetic data experiments on div2k?

**Comment by DizzyYunxuan** [comment]
Our HR data (target domain) and real-world LR (source domain) are unpaired. The number of HR and LR are not have to be equal. 
You can set any HR data(target domain) as you wish.
